### PR TITLE
Add Kubernetes functionality to droplet kit

### PR DESCRIPTION
**Pull request by scottcrawford03**
This adds all the api wrappers for both clusters and node pools for `kubernetes`.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+### Version 2.6.2
```
Comment (hilary) [inline CHANGELOG.md]: Version 2.7.0 

Unless I'm missing something, this is a minor release. From [Semantic Versioning 2.0.0](https://semver.org/#semantic-versioning-200)

> MINOR version when you add functionality in a backwards-compatible manner

File: README.md [hunk 2]
```diff
@@ -288,6 +288,40 @@ Image Actions Supported:
...
+## Kubernetes Resource
+
+```
+client = DropletKit::Client.new(access_token: 'TOKEN')
+client.kubernetes_clusters #=> DropletKit::KubernetesClusterResource
+```
+
+Actions supported
+
+When `cluster` is refered to in some of the arguments below, it is referring to:
```
Comment (hilary) [inline README.md]: When the arguments below refer to `cluster`, they refer to:

File: README.md [hunk 3]
```diff
@@ -288,6 +288,40 @@ Image Actions Supported:
...
+```
+cluster = DropletKit::KubernetesCluster.new(name: "foo", region: "nyc1", ...) # cluster attributes
+```
+
+When `node_pool` is refered to in some of the arguments below, it is referring to:
```
Comment (hilary) [inline README.md]: When the arguments below...

File: lib/droplet_kit/client.rb [hunk 4]
```diff
@@ -1,4 +1,5 @@
 require 'faraday'
+require 'pry'
```
Comment (hilary) [inline lib/droplet_kit/client.rb]: Oops! Can't be here 🙁

File: lib/droplet_kit/mappings/kubernetes_options_mapping.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
+module DropletKit
+  class KubernetesOptionsMapping
+
```
Comment (hilary) [inline lib/droplet_kit/mappings/kubernetes_options_mapping.rb]: remove blank line

File: lib/droplet_kit/mappings/kubernetes_node_mapping.rb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+module DropletKit
+  class KubernetesNodeMapping
+    include Kartograph::DSL
+
```
Comment (hilary) [inline lib/droplet_kit/mappings/kubernetes_node_mapping.rb]: remove blank line

File: lib/droplet_kit/mappings/kubernetes_node_pool_mapping.rb [hunk 7]
```diff
@@ -0,0 +1,28 @@
+module DropletKit
+  class KubernetesNodePoolMapping
+    include Kartograph::DSL
+
```
Comment (hilary) [inline lib/droplet_kit/mappings/kubernetes_node_pool_mapping.rb]: remove blank line

File: lib/droplet_kit/mappings/kubernetes_options_mapping.rb [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    include Kartograph::DSL
+
```
Comment (hilary) [inline lib/droplet_kit/mappings/kubernetes_options_mapping.rb]: remove blank line

File: lib/droplet_kit/models/kubernetes_options.rb [hunk 9]
```diff
@@ -0,0 +1,9 @@
+module DropletKit
+  class KubernetesOptions < BaseModel
+
```
Comment (hilary) [inline lib/droplet_kit/models/kubernetes_options.rb]: remove blank line before and after attributes

File: lib/droplet_kit/version.rb [hunk 10]
```diff
@@ -1,3 +1,3 @@
 module DropletKit
-  VERSION = "2.6.1"
+  VERSION = "2.6.2"
```
Comment (hilary) [inline lib/droplet_kit/version.rb]: VERSION = "2.7.0" as above

File: lib/droplet_kit/client.rb (hunk 4 above)
Comment (scottcrawford03) [reply lib/droplet_kit/client.rb]: womp womp

File: CHANGELOG.md (hunk 1 above)
Comment (scottcrawford03) [reply CHANGELOG.md]: 👍 still learning semantic lyfe. will update.

**Pull request merged**